src/azure-cli/azure/cli/command_modules/ams/tests/latest/test_ams_transform_scenarios.py

```diff
@@ -7,7 +7,7 @@
 
 from azure.cli.core.util import CLIError
 from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer, StorageAccountPreparer
-from azure.cli.command_modules.ams._test_utils import _get_test_data_file
+from ._test_utils import _get_test_data_file
```

Comment:
Just curious, why change the absolute path to a relative path?

Reply:
The `_test_utils` file is under the source code directory before (azure/cli/command_modules/ams), which is not correct. It's only been used in test files. So I move it to azure/cli/command_modules/ams/tests/latest. In the test files, we do not need to use absolute path as pyinstaller will not resolve test files. It will be ugly if we use `azure.cli.command_modules.ams.tests.latest._test_utils` here.

Reply:
Ok, I see.